conn_test.go

```diff
@@ -195,14 +212,152 @@ func TestConnectionPool_Broadcast(t *testing.T) {
 		},
 	}
 
-	cp.Broadcast(msg)
+	cp.BroadcastSingleMessage(msg)
+
+	// wait until receive the msg
+	wg.Wait()
+}
+
+func Execute_BroadcastMultipleMessage(textList []string, waitSize int) ([]bool, error) {
+	cp := cleisthenes.NewConnectionPool()
```

Comment:
### change function name into camel-case & private function

### refactor large function

I think what this function do is 

1. make connection list
2. register mock handler for each connection
3. ...

I think we can make function prettier & clear